linkerd/app/integration/tests/telemetry.rs

```diff
@@ -751,7 +751,7 @@ mod transport {
         drop(client);
         assert_eventually_contains!(
             metrics.get("/metrics"),
-            "tcp_close_total{direction=\"inbound\",peer=\"src\",tls=\"disabled\",errno=\"\"} 1"
+            "tcp_close_total{direction=\"inbound\",peer=\"src\",tls=\"disabled\",errno=\"ENOTCONN\"} 1"
```

Comment:
This appears to be the main functional change of moving the metrics layer.

Reply:
Seems good to me!